blog/2020-07-01-LetsOptaPlanYourJbpmTasksPart1.adoc

```diff
@@ -0,0 +1,215 @@
+= Let's OptaPlan your jBPM tasks (part 1) - Integrating the two worlds
+:page-interpolate: true
+:awestruct-author: wmedvede
+:awestruct-layout: blogPostBase
+:awestruct-tags: [feature, task assigning, bpm integration]
+
+:awestruct-share_image_filename: CreditDisputeProcessDiagram.png
+
+Traditionally, business process management systems (BPMS) assign human tasks to users on a group basis.
+This information is part of the human tasks configuration.
+
+In the following process, we can see how the Resolve Dispute and Notify Customer tasks are configured with the
+CreditAnalyst and ClientRelations groups respectively:
+
+image::CreditDisputeProcessDiagram.png[]
+
+At runtime, every time a new instance of one of these tasks is created it is automatically assigned to the respective
+group with the status Ready.
+
+This makes the task automatically available to all users that belong to the group.
+All users in the group  see it in their respective *Task Inbox* but none are the actual owner or responsible for the task.
+To be able to work with a particular task, a user must execute the claim operation to reserve the task for that user.
+This is a completely manual procedure.
+
+The following image shows a list of tasks with the Ready status:
+
+image::TaskList-NoTaskIsSelected.png[]
+
+The following image shows the result of the claim operation executed on task #74 by the user katy:
```

Comment:
(soft suggestion)
Wrap katy in backticks (`) or italic (_) to signal it's data on screen.